src/TitleTests.php

```diff
@@ -530,8 +532,7 @@ function show_results_summary($post_id) {
      * @return bool
      */
     function is_single() {
-        $is_single = (is_single() && get_post_type() == 'post');
-        return apply_filters('split_tests_is_single', $is_single);
+        return apply_filters('split_tests_is_single', is_single());
```

Comment:
Removing `get_post_type()` from this helper is what fixed the URL slug selection logic for test posts.